packages/resolver-svimg/package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Let's keep this ESLint PR branched off `main` so that it's not dependent on any open PRs or experimental branches